<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Fixed Size Font editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lblFontTitle">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Font Symbols</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listFontGlyphs">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lblCurrentSymbolIndex">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFontGlyphEditor" name="widgetGlyphEditor" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionLoadROMImageFont"/>
    <addaction name="actionLoadPSF"/>
    <addaction name="actionSave_font_to_ROM_image"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExitApp"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoadROMImageFont"/>
   <addaction name="actionLoadPSF"/>
   <addaction name="actionSave_font_to_ROM_image"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionCopy_glyph"/>
   <addaction name="actionCut_glyph"/>
   <addaction name="actionPaste_glyph"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExitApp">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit aplication</string>
   </property>
  </action>
  <action name="actionLoadROMImageFont">
   <property name="icon">
    <iconset resource="psfeditor.qrc">
     <normaloff>:/images/open_mif.png</normaloff>:/images/open_mif.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load font from ROM image</string>
   </property>
  </action>
  <action name="actionSave_font_to_ROM_image">
   <property name="icon">
    <iconset resource="psfeditor.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save font ...</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="psfeditor.qrc">
     <normaloff>:/images/save_as.png</normaloff>:/images/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;font as ...</string>
   </property>
   <property name="toolTip">
    <string>Save the font into another file</string>
   </property>
  </action>
  <action name="actionCopy_glyph">
   <property name="icon">
    <iconset resource="psfeditor.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy glyph ...</string>
   </property>
   <property name="toolTip">
    <string>Copy glyph to clipboard ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut_glyph">
   <property name="icon">
    <iconset resource="psfeditor.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut glyph ...</string>
   </property>
   <property name="toolTip">
    <string>Cut glyph and move to clipboard ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste_glyph">
   <property name="icon">
    <iconset resource="psfeditor.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste glyph ...</string>
   </property>
   <property name="toolTip">
    <string>Paste current glyph from clipboard ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionLoadPSF">
   <property name="icon">
    <iconset resource="psfeditor.qrc">
     <normaloff>:/images/open_psf.png</normaloff>:/images/open_psf.png</iconset>
   </property>
   <property name="text">
    <string>Load font from &amp;PSF file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QFontGlyphEditor</class>
   <extends>QWidget</extends>
   <header>qfontglypheditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="psfeditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listFontGlyphs</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_listFontGlyphs_itemSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_listFontGlyphs_itemSelectionChanged()</slot>
 </slots>
</ui>
